accounts_backend/explores/mozilla_accounts_multi_service_dau.explore.lkml

```diff
@@ -0,0 +1,18 @@
+include: "../views/mozilla_accounts_multi_service_dau.view.lkml"
+
+explore: mozilla_accounts_multi_service_dau {
+  label: "Multi-Service Active Mozilla Accounts"
+
+  conditionally_filter: {
+    filters: [date_date: "30 days ago for 30 days"]
+    unless: [date_week, date_month, date_quarter, date_year]
```

Comment:
I just recently learned that if you conditionally filter one of the dimensions from a time dimension group then Looker automatically treats the other dimensions from that time dimension group as if they're specified in `unless` ([docs](https://cloud.google.com/looker/docs/reference/param-explore-conditionally-filter#conditionally_filter_with_a_dimension_of_type_time_in_a_group_puts_the_groups_other_dimensions_in_the_unless_subparameter)), so you could remove the `unless` here and it would work the same.

That said, I'm also fine with you keeping the `unless` as is since it makes explicit what's happening.